addons/web/static/src/scss/utilities_custom.scss

```diff
@@ -42,7 +42,15 @@ $order-array: (
 }
 
 
-$utilities: () !default;
+$utilities: (
+    "text": (
+        class: text,
+        property: white-space,
+        values: (
+            prewrap: pre-wrap,
+        ),
+    ),
+) !default;
```

Comment:
Actually, this will not work...

Look at the "display" below as an example :wink:

Reply:
Done! :slightly_smiling_face: